### PR TITLE
Add form

### PR DESCRIPTION
**Pull request by zbahati**
# Add form 

- Add controllers for comments
- add action to the comments controller
- Add likes Controller
- Add action to the likes controller
- Test if is working correctly

### CODE REVIEW

File: app/controllers/likes_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @like = Like.new(user_id: @user.id, post_id: @post.id)
+
+    redirect_to user_post_path(@user, @post, @like)
+  end
```
Comment (aamirkhan2478) [inline app/controllers/likes_controller.rb]: - [ ] Please check your like functionality it's not working kindly fix it check below :point_down:  my suggestion to fix this issue :slightly_smiling_face: 

```suggestion
  def create
    @user = current_user
    @post = Post.find(params[:post_id])
    @like = Like.new(user_id: @user.id, post_id: @post.id)

     if @like.save
      redirect_to user_post_path(@user, @post)
    end
  end
```

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @zbahati, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 1 above)
Comment (Whoistolu) [reply app/controllers/likes_controller.rb]: - Well resolved

**Review by Whoistolu — APPROVED** [review]
### Hi @zbahati 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**